style.css

```diff
@@ -96,3 +121,182 @@ header {
   width: 1.5rem;
   height: 1.5rem;
 }
+
+#Contact-me {
+  background: #6070ff;
+  border-top-left-radius: 80px;
+  background-image: url("/image/headerbg.png");
+  background-size: cover;
+  background-position: 100% 10%;
+  background-repeat: no-repeat;
+  height: auto;
+  width: 100%;
+  padding: 16px;
+}
+
+.contact {
+  width: 327px;
+  height: 102px;
+  margin: auto;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.contact-title {
+  margin: 70px auto 16px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  padding: 1rem 0 0;
+  text-align: center;
+  color: #fff;
+}
+
+.container {
+  padding-top: 5rem;
+}
+
+.text-field {
+  height: 48px;
+  width: 327px;
+  margin: 0;
+  text-align: center;
+}
+
+#fname {
+  height: 48px;
+  width: 340px;
+  border-radius: 8px;
+}
+
+#lname {
+  margin-top: 1rem;
+  height: 48px;
+  width: 340px;
+  border-radius: 8px;
+}
+
+.subject {
+  margin: 0;
+}
+
+#subject {
+  border-radius: 8px;
+  height: 100px;
+  width: 340px;
+  margin-top: 2rem;
+  color: #b3bac5;
+}
+
+button {
+  width: 140px;
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 10px;
+  border: 1px solid #dfe1e6;
+  text-align: center;
+  letter-spacing: 0.03em;
+  margin: auto;
+}
```

Comment:
**[OPTIONAL]** - Your buttons aren't styled correctly according to the Figma template. Please try to fix this for all required buttons. Note that it is vital to replicate the design template as closely as possible to give confidence to recruiters in our ability to replicate exact design specs. 👍 👍 . Don't forget to implement the styling for interactions such as `hover`

Your project:
![button project](https://user-images.githubusercontent.com/73607512/192119605-835ce643-db4a-4dda-83c5-0ce0d75de8f0.PNG)

Figma Design:
![buttons figma](https://user-images.githubusercontent.com/73607512/192119614-3a7b816e-07b9-4c69-b12d-22cfcb17b5a6.PNG)